src/io/eventloop.c

```diff
@@ -292,11 +292,11 @@ void MVM_io_eventloop_destroy(MVMThreadContext *tc) {
 
     if (instance->event_loop) {
         uv_close((uv_handle_t*)instance->event_loop_wakeup, NULL);
-        MVM_free(instance->event_loop_wakeup);
         instance->event_loop_wakeup = NULL;
 
         /* Not sure we can always do this */
         uv_loop_close(instance->event_loop);
+        MVM_free(instance->event_loop_wakeup);
```

Comment:
I think the `instance->event_loop_wakeup = NULL;` would also need to move; otherwise, we set it to NULL, and then do `MVM_free(NULL)`, which silently does nothing.

Reply:
Sorry about that. Should be fixed now.